build/.vsts-PRInternalChecks-azureBuild-pipeline.yml

```diff
@@ -0,0 +1,150 @@
+name: pr$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:-r)
+parameters:
+- name: BuildConfiguration
+  default: release
+- name: NugetSecurityAnalysisWarningLevel
+  default: warn   
+variables:
+- name: vmImage
+  value: windows-2019
```

Comment:
Nit: Could this be changed to windows-latest. That way we don't need to update it in the future.

Reply:
Changed to windows-latest.